openpype/hosts/maya/plugins/publish/collect_arnold_scene_source.py

```diff
@@ -21,11 +22,13 @@ def process(self, instance):
                 self.log.warning("Skipped empty instance: \"%s\" " % objset)
                 continue
             if objset.endswith("content_SET"):
-                instance.data["setMembers"] = cmds.ls(members, long=True)
-                self.log.debug("content members: {}".format(members))
+                set_members = get_all_children(cmds.ls(members, long=True))
+                instance.data["setMembers"] = set_members
+                self.log.debug("content members: {}".format(set_members))
             elif objset.endswith("proxy_SET"):
-                instance.data["proxy"] = cmds.ls(members, long=True)
-                self.log.debug("proxy members: {}".format(members))
+                set_members = get_all_children(cmds.ls(members, long=True))
+                instance.data["proxy"] = set_members
+                self.log.debug("proxy members: {}".format(set_members))
```

Comment:
Sorry, not entirely sure what you mean?

Reply:
I meant to say to also not include all children here like with `setMembers` but since you renamed the other data to keep the logic like this you can ignore this comment.